db/import-tmpl/status.yml

```diff
@@ -37,8 +37,9 @@
 #   Fail:                     List of Status Changes that causes the status to fail to activate. (Optional)
 #   End:                      List of Status Changes that will end when the status activates. (Optional)
 #   EndReturn:                List of Status Changes that will end when the status activates and won't give its effect. (Optional)
+#   Cancel:                   List of Status Changes that will be canceled when the status becomes inactive. (Optional)
```

Comment:
Cancel somewhat sounds ambigous.
People that dont read closely will think "End" and "Cancel" is the same.
Dont really have a better suggestion though. :-(

Reply:
Agreed, I thought about the confusion as I was working on this. I'll see if I can come up with a better label name.